eva-peng/week5/tues/players/app/views/players/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Add New Player</h2>
+
+<%= form_with model: @player, local: true do |f| %>
+
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+    <br>
+
+    <%= f.label :sex %>
+    <%= f.text_field :sex %>
+    <br>
+
+    <%= f.label :dob %>
+    <%= f.text_field :dob %>
+    <br>
+
+    <%= f.label :grand_slam %>
+    <%= f.text_field :grand_slam %>
+    <br>
+
+    <%= f.label :highest_ranking %>
+    <%= f.text_field :highest_ranking %>
```

Comment:
In your `schema.rb`, it has integer as the field, but you have specified a text field here - please make them the same
 ` t.integer "grand_slam"`
` t.integer "highest_ranking"`